developer_manual/app/hooks.rst

```diff
@@ -156,7 +156,9 @@ Filesystem hooks available in scope **\\OC\\Files**:
 
 Filesystem Scanner
 ------------------
-Filesystem scanner hooks available in scope **\\OC\\Utils\\Scanner**:
+Filesystem scanner hooks available in scope **\\OC\\Files\\Utils\\Scanner**:
 
 * **scanFile** (string $absolutePath)
 * **scanFolder** (string $absolutePath)
+* **postScanFile** (string $absolutePath)
```

Comment:
If you want a highlighted note (like the blue box near the top of https://doc.owncloud.org/server/8.2/admin_manual/installation/linux_installation.html ) mark it up like this:

````  .. note:: This is available since ownCloud versions 8.1.4 and 8.2.0 ````

Is that what you're looking for?